Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/day/tabs/location/ContactDiaryLocationListViewModel.kt

```diff
@@ -50,18 +70,21 @@ class ContactDiaryLocationListViewModel @AssistedInject constructor(
                 )
             )
         } else {
-            val visit = dayElement.first()
+            val visit = dayElement
+                .first()
                 .find { it.contactDiaryLocation.locationId == item.item.locationId }
             visit?.let { contactDiaryRepository.deleteLocationVisit(it) }
         }
     }
 
-    companion object {
-        private val TAG = ContactDiaryLocationListViewModel::class.java.simpleName
-    }
-
     @AssistedInject.Factory
     interface Factory : CWAViewModelFactory<ContactDiaryLocationListViewModel> {
         fun create(selectedDay: String): ContactDiaryLocationListViewModel
     }
 }
+
+private val TAG = ContactDiaryLocationListViewModel::class.java.simpleName
+private const val SELECTED_CONTENT_DESCRIPTION = R.string.accessibility_location_selected
+private const val UNSELECTED_CONTENT_DESCRIPTION = R.string.accessibility_location_unselected
+private const val SELECT_ACTION_DESCRIPTION = R.string.accessibility_action_select
+private const val DESELECT_ACTION_DESCRIPTION = R.string.accessibility_action_deselect
```

Comment:
R.string.* are actually constants, any reason why it should be assigned to another constant?

Reply:
I do it because it is easier to change/ maintain